src/Jira/Api.php

```diff
@@ -157,14 +157,14 @@ public function getAttachment($attachmentId)
         return $result;
     }
 
-    public function getProjects()
+    public function getProjects($expand = '')
     {
-        return $this->api(self::REQUEST_GET, "/rest/api/2/project");
+        return $this->api(self::REQUEST_GET, "/rest/api/2/project",array('expand' => $expand), true);
```

Comment:
The `, true` is setting `$return_as_json` to `true` and therefore making return value just an array instead of `Result` class. This is a BC break.

Would you please explain why you did this?